cdap-ui/app/cdap/components/DataPrep/Directives/ExtractFields/UsingPatternsModal/index.js

```diff
@@ -423,24 +442,17 @@ export default class UsingPatternsModal extends Component {
             }
           </div>
         </ModalBody>
-        <ModalFooter>
-          <span className="text-warning">
-            {this.state.error}
-          </span>
-          <button
-            className="btn btn-primary"
-            onClick={this.applyDirective}
-            disabled={isNil(this.state.pattern) ? 'disabled' : null}
-          >
-            {T.translate('features.DataPrep.Directives.ExtractFields.extractBtnLabel')}
-          </button>
-          <button
-            className="btn btn-secondary"
-            onClick={this.props.onClose}
-          >
-            {T.translate('features.DataPrep.Directives.cancel')}
-          </button>
-        </ModalFooter>
+        {
+          this.state.error ?
+            <CardActionFeedback
+              type="DANGER"
+              message={this.state.error}
+            />
+          :
+            <ModalFooter>
+              {this.renderFooterButton()}
+            </ModalFooter>
+        }
```

Comment:
If there is an error and the `CardActionFeedback` is active, the user cannot click on the button anymore?

Reply:
Hhmm this is the pattern we have today. Might be good to show errors and allow users to make changes and apply the directive again.